javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,49 @@
+const width = 6;
+const height = 12;
+
+const area = width * height;
+
+console.log('value of area is:', area);
+console.log('typeof area is:', typeof area);
+
+const bill = 150;
+const payment = 500;
+
+const change = bill - payment;
+
+console.log('value of change is:', change);
+console.log('typeof change is:', typeof change);
+
+const quizzes = 50;
+const midterm =  150; 
+const final = 200;
+
+const grade = (quizzes + midterm + final) / 3;
+
+console.log('value of grade is:', grade);
+console.log('typeof grade is:', typeof grade);
+
+const firstName = "Dennis";
+const lastName = "Nguyen";
+let fullName = firstName + ' ' + lastName;
```

Comment:
This should be `const`. `fullName` never gets reassigned, so it's value will always stay the same, so it should be declared using `const`